### PR TITLE
Clients and communities

### PR DESCRIPTION
**Pull request by scrool**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -57,8 +57,62 @@ License
 
 Documentation is available under MIT license.
 
+Client implementations
+----------------------
+
+There multiple clients, libraries or just small projects that can be used to control Twinkly devices:
+
+* `xled`_  - python library and command line interface
+* `xled_plus`_ - Addons to the XLED package, to create nice effects for Twinkly LED lights
```
Comment (scrool) [inline README.rst:66]: @Anders-Holst how would you like your project to be named here? On GitHub it is called `xled_plus`, project on PyPI is `xled-plus` and in the text you call it `XLED Plus`.

File: README.rst [hunk 2]
```diff
@@ -57,8 +57,62 @@ License
...
+* `twinkly-animator`_ - Drawing and animation tools for Twinkly LED lights
+* Python class to interact with generation I device and IDA Pro loader of firmware binary in `Twinkly Twinkly Little Star by F-Secure LABS`_.
+
+Communities
+-----------
+
+There are multiple places where you can connect with others who are interested in Twinkly devices:
+
+* `Twinkly (unofficial users group) on Facebook`_ to share your installations, show your creations, ask questions, post some hacks. Anything Twinkly related goes.
+* `Discussions on GitHub in xled_plus project`_ with daily effects.
```
Comment (scrool) [inline README.rst:91]: @Anders-Holst and I guess here it makes a sense to call it by GitHub project, right?

**Comment by scrool** [comment]
This will be rendered like this https://xled-docs--28.org.readthedocs.build/en/28/

File: README.rst (hunk 2 above)
Comment (Anders-Holst) [reply README.rst:91]: Yes! It looks nice. Also the reference under Client Implementations looks good. Thanks!

**Pull request merged**